pages.ml/common/cargo.md

```diff
@@ -0,0 +1,37 @@
+# cargo
+
+> റസ്റ്റ് പ്രോജക്റ്റുകളും അവയുടെ മൊഡ്യൂൾ ഡിപൻഡൻസികളും (ക്രാറ്റുകൾ) കൈകാര്യം ചെയ്യുക.
+> `ബിൽഡ്` പോലുള്ള ചില ഉപകമാൻഡുകൾക്ക് അവരുടേതായ ഉപയോഗ ഡോക്യുമെന്റേഷൻ ഉണ്ട്.
```

Comment:
Currently, we don't have a Malayalam translation template for the subcommand, can I add this here https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#ml (or would it change based on context)?